### DISCUSSION TITLE
"None of the genes were in the PIN"

### DISCUSSION THREAD

**Discussion by Jmperez2080** [egeulgen/pathfindR#113] [Q&A] [answered]
Hi,

I'm looking to do a pathway analysis on a proteomics data set (where I compare treatment vs. control). I have my "gene symbols" as protein accession numbers. Is there a PIN I can download that corresponds to protein accession numbers (organism: homo sapiens)? 

If not, is there a better way to use pathfindR with proteomics datasets? 

Thank you!

**Comment by egeulgen** [comment, marked as answer]
Hey @Jmperez2080,

I would suggest that the easiest way is to convert your identifiers into HGNC symbols. For this, you can use Ensembl's [BioMart](https://www.ensembl.org/biomart/martview) or the R package for it: [biomaRt](https://bioconductor.org/packages/biomaRt).

Best,
-E
